### PR TITLE
Update Media Player

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: pages/media/index.vue [hunk 1]
```diff
@@ -51,6 +53,7 @@
         script: [
           { src: `https://code.jquery.com/jquery-${this.JQUERY_VERSION}.min.js` },
           { src: `https://code.jquery.com/ui/${this.JQUERY_UI_VERSION}/jquery-ui.min.js` },
+          { src: this.dashRequired ? `https://unpkg.com/dashjs@${this.DASHJS_VERSION}/dist/dash.all.min.js` : '' },
```
Comment (lbiedinger) [inline pages/media/index.vue:56]: This results in an empty <src> tag being added to the markup.
Try this instead:
```
        script: [
          { src: `https://code.jquery.com/jquery-${this.JQUERY_VERSION}.min.js` },
          { src: `https://code.jquery.com/ui/${this.JQUERY_UI_VERSION}/jquery-ui.min.js` },
          { src: `https://unpkg.com/@europeana/media-player@${this.MEDIA_PLAYER_VERSION}/dist/europeana-media-player.min.js` }
        ].concat(this.dashRequired ? [
          { src: `https://unpkg.com/dashjs@${this.DASHJS_VERSION}/dist/dash.all.min.js` }
        ] : [])
```

File: plugins/media.js [hunk 2]
```diff
@@ -73,3 +73,7 @@ export function isRichMedia(media) {
     isIIIFMedia(media) ||
     isPlayableMedia(media);
 }
+
+export function requiresDashJS(media) {
```
Comment (lbiedinger) [inline plugins/media.js:77]: Could you add a unit test?
Should be straight forward, there are already similar ones in `tests/unit/plugins/media.spec.js` for the rest of the methods in here.

File: pages/media/index.vue (hunk 1 above)
Comment (alansutherland) [reply pages/media/index.vue:56]: thanks I knew there was better way of doing this, just couldn't think of it

File: plugins/media.js (hunk 2 above)
Comment (alansutherland) [reply plugins/media.js:77]: sure

File: components/item/MediaPresentation.vue [hunk 3]
```diff
@@ -17,7 +17,7 @@
       <iframe
         data-qa="media player"
         allowfullscreen="true"
-        :src="$path({ name: 'media', query: { id: europeanaIdentifier, mediaUrl: media.about } })"
+        :src="$path({ name: 'media', query: { id: europeanaIdentifier, mediaUrl: media.about, dashRequired: requiresDashJS } })"
```
Comment (rwd) [inline components/item/MediaPresentation.vue:20]: Remove this and replace with `mediaType: media.ebucoreHasMimeType`. Then make the media page responsible for performing the check against `requiresDashJS`.

Comment (alansutherland) [reply components/item/MediaPresentation.vue]: ok thanks, have updated

**Pull request merged**